scripts/jenkins/ardana/ansible/roles/ardana_deploy/tasks/run_lint.yml

```diff
@@ -0,0 +1,32 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- name: Find ansible files under "{{ lint_path }}"
+  command: >
+    find "{{ lint_path }}" -type f
+    -name _ardana-base*yml -or -name client*yml -or
+    -name *ceilometer*.yml -or -name *FND-AP2*.yml -or
+    -name *FND-CLU*.yml -or -name *glance*.yml -or
+    -name *heat*.yml -or -name *magnum*.yml -or
+    -name *monasca-transform*.yml -or -name *nova*.yml -or
+    -name *percona*.yml -or -name *rabbitmq*.yml -or
+    -name *spark*.yml -or -name *tls*.yml
+  register: _lint_files
+  #TODO: run on all yml files. The current file list is copied from
+  # ardana-dev-tools, and any other files are not yet expected to pass linting.
+
+- name: Run ansible-lint on the files found
+  command: "{{ lint_venv }}/bin/ansible-lint {{ lint_params }} {{ _lint_files.stdout }}"
```

Comment:
So we likely need to run against a copy of the scratch area, as some quick testing against the ~/openstack/ardana/ansible tree failed with errors about references to non-existing files that are generated when creating the scratch area:
```
ardana@deployer:/tmp/ardana.ansible.fergal> $alint -p *.yml
Traceback (most recent call last):
  File "/tmp/ansible-lint/bin/ansible-lint", line 110, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/tmp/ansible-lint/bin/ansible-lint", line 97, in main
    matches = runner.run()
  File "/tmp/ansible-lint/lib/python2.7/site-packages/ansiblelint/__init__.py", line 180, in run
    skip_list=set(self.skip_list)))
  File "/tmp/ansible-lint/lib/python2.7/site-packages/ansiblelint/__init__.py", line 106, in run
    with open(playbookfile['path'], 'Ur') as f:
IOError: [Errno 2] No such file or directory: u'/tmp/ardana.ansible.fergal/ardana-stop.yml'
```

Testing against a copy of the generated scratch area took approx 10 mins to run, and yielded the sort of output I would expect...

Reply:
Are you saying that we should **only** run against the scratch area because `~openstack/ardana/ansible` will always reference files that have not yet been created?